includes/admin/class-sensei-learners-main.php

```diff
@@ -445,11 +446,11 @@ protected function get_row_data( $item ) {
 							'sensei-learner-action-enrol'
 						);
 
-						$actions[] = '<a class="learner-action button" data-user-id="' . esc_attr( $user_activity->user_id ) . '" data-action="enrol" href="' . esc_url( $enrol_action_url ) . '">' . esc_html__( 'Manually enroll learner', 'sensei-lms' ) . '</a>';
+						$row_actions[] = '<span><a class="learner-action" data-user-id="' . esc_attr( $user_activity->user_id ) . '" data-action="enrol" href="' . esc_url( $enrol_action_url ) . '">' . esc_html__( 'Enroll', 'sensei-lms' ) . '</a></span>';
 					} else {
 						$manual_enrolment_disabled_reason = __( 'Enrollment is given by another provider.', 'sensei-lms' );
 
-						$actions[] = '<a class="learner-action button disabled" disabled="disabled" title="' . esc_attr( $manual_enrolment_disabled_reason ) . '">' . esc_html__( 'Manually enroll learner', 'sensei-lms' ) . '</a>';
+						$row_actions[] = '<span><a class="learner-action disabled" disabled="disabled" title="' . esc_attr( $manual_enrolment_disabled_reason ) . '">' . esc_html__( 'Enroll', 'sensei-lms' ) . '</a></span>';
```

Comment:
I think we can skip showing the `Enroll` action here when they are already enrolled.

Reply:
Hum, I agree with you. But not sure if it was added to make more explicit that it was given by another provider. @donnapep WDYT about that?

Reply:
Yup it made sense for manual enrollment, but we are no longer wording it explicitly as manual, right? If this is just `Enroll`, having a disabled `Enroll` in just some cases can be confusing. (As we are not showing the action where they have manual enrolment)

Reply:
👍 Updated here: https://github.com/Automattic/sensei/pull/3404/commits/1ac5bc1f7dfc2bf4f981c240329d017795a2a9c1